runtime/src/append_vec.rs

```diff
@@ -497,7 +498,7 @@ pub mod tests {
 
         assert_eq!(dav.get_account_test(index2).unwrap(), account2);
         assert_eq!(dav.get_account_test(index1).unwrap(), account1);
-        std::fs::remove_file(path).unwrap();
+        std::fs::remove_file(&path.path).unwrap();
```

Comment:
this should be unnecessary, is done by implicit drop(path)...

Reply:
the account store needs to be removed so that an empty store could get created on deserialize, changed it to drop instead remove

Reply:
:thumbsup: